components/cpu/metrics/metrics.go

```diff
@@ -98,7 +104,7 @@ func SetLoadAverage(ctx context.Context, duration time.Duration, avg float64, cu
 }
 
 func SetUsedPercent(ctx context.Context, pct float64, currentTime time.Time) error {
-	usedPercent.WithLabelValues("cpu").Set(pct)
+	usedPercent.With(prometheus.Labels{}).Set(pct)
```

Comment:
Do we have to pass an empty labels to the function?

Reply:
Yes, the type in vector